src/components/SeasonCard.tsx

```diff
@@ -33,20 +36,24 @@ const SeasonCard: React.FC<SeasonCardProps> = ({ details }): JSX.Element => {
                         ? `https://image.tmdb.org/t/p/w500${details.poster_path}`
                         : '/poster-placeholder.jpeg'
                 }
-                alt={details.name}
+                alt={details.name + ' poster'}
             />
 
-            <div className='flex flex-col max-h-[270px] p-2 sm:p-4 max-w-[1200px]'>
+            <div className='flex flex-col max-h-[270px] p-2 sm:p-4'>
                 <Typ
                     className='sm:text-left sm:py-1 hover:text-blue-500 cursor-pointer'
                     fontWeight={'bold'}
                 >
                     {details.name}
                 </Typ>
                 <div className='flex justify-around sm:justify-start sm:py-1'>
-                    <Typ className='hidden sm:inline pr-4'>{details.vote_average}</Typ>
-                    <Typ className='sm:px-4'>{details.air_date.slice(0, 4)}</Typ>
-                    <Typ className='sm:px-4'>{details.episode_count} Episodes</Typ>
+                    {details.season_number != 0 && details.air_date && (
+                        <Typ className='hidden sm:inline pr-4'>{details.vote_average}</Typ>
+                    )}
+                    <Typ className='sm:pr-4'>
+                        {details.air_date ? details.air_date.slice(0, 4) : '-'}
+                    </Typ>
+                    <Typ className='sm:pr-4'>{`${details.episode_count} ${pluralizeString(details.episode_count, 'Episode')}`}</Typ>
```

Comment:
I think this section is a little hard to read. I like all the content but think it could be improved with a few things.
1. maybe the rating could have a star next to it or something, it's not totally clear what it is. It could also be on it's own line.
2. Some sort of separator between the date and episode count. "|" "-" or "•"